### DISCUSSION TITLE
Expanding the permissions I have given myself in my app

### DISCUSSION THREAD

**Discussion by peter2454** [saleweaver/python-amazon-sp-api#1113] [General]
Hi - Newbie question here -

I have just expanded the permissions I have given my private app by going into my Developer Profile in Seller Central, and ticking more boxes there.  I also expanded permissions from the 'Edit App' button and ticked more options there.

I have received email from Amazon to say these changes have been approved.

However I don't seem at the moment to be able to access more areas of the API than I did before. Do I need to give it a certain amount of time for the changes to actually come into effect, or is there something else I must do to make the changes I made in my Developer Profile come into effect?

Many thanks in advance for your help on this!

**Comment by saleweaver** [comment]
Always change the refresh token. Make sure to check the docs, this is all explained.
